chef/cookbooks/provisioner/templates/default/crowbar_join.suse.sh.erb

```diff
@@ -132,14 +132,19 @@ sync_time() {
 
     local tries_left=120
 
-<% if @target_platform_version.to_f < 12.0 %>
-    SNTP_OPTS="-P no -r"
-<% else %>
-    SNTP_OPTS="-s"
-<% end %>
+    # ntpd 4.2.4 (e.g. on SLES11-SP3 GA) didn't contain a working ntpdate. Use sntp
+    # there. ntpdate got reintroduced with a newer ntpd versions (4.2.8x) where the
+    # sntp commandline interface changed in an incompatible way. Use ntpdate on
+    # such systems (i.e. SLES-11-SP3 with updates and SLES12)
+    if ntpd --version | grep -q "4\.2\.4"; then
+        ntp="sntp -P no -r"
+    else
+        ntp="/usr/sbin/ntpdate"
```

Comment:
> We use /usr/sbin/ntpdate -u in Cloud 6 (to avoid the the NTP socket is in use, exiting issue), should we do the same here?

Yeah, I'll add that.

> Btw, in Cloud 6, we pass all ntp servers at once to ntpdate, instead of doing a loop for each of them.

AFAICS the old sntp does not allow passing more than one server at a time :/